sources/install/package_base.sh

```diff
@@ -396,6 +396,16 @@ function post_install() {
     if [ -d "/root/Desktop" ]; then rm -r /root/Desktop; fi
 }
 
+function uploader() {
+    colorecho "Installing Uploader"
+    git clone https://github.com/Frozenka/uploader.git /opt/tools/uploader
+    pip3 install -r /opt/tools/uploader/requirements.txt
+    add-aliases uploader
+    add-history uploader
+    add-test-command "uploader.py --help"
```

Comment:
You refer to `uploader.py` but your alias file refers to the `uploader` command.

Reply:
done